[DOCS-8074] Add Obs Pipelines Monitoring doc (#25263)
DataDog/documentation: content/en/observability_pipelines/monitoring/_index.md

- You can also create your own dashboards, notebooks, and monitors with the available [Observability Pipelines metrics][5].

## View the status of your pipelines

1. Navigate to [Observability Pipelines][1] to see how many bytes your pipelines are receiving and sending out.
1. Select your pipeline.
1. Click the **Health** tab to see details about the pipeline and its components. Specifically, view graphs of:
    - How much each component is being utilized and the total number of events that the component receives and sends out.
    - The number of requests made to destinations and the number of errors encountered.
    - How many events are intentionally and unintentionally discarded.
    - Any changes over the week with the number of requests and errors for each component.

You can export a health graph to a dashboard, notebook, or monitor. The exported graph shows you how the metric is grouped by the specific pipeline and component tags.

## View the status of your Workers

To view graphs of your Observability Pipelines Workers' resource utilization and the amount of data sent through:

1. Navigate to [Observability Pipelines][1].
1. Select your pipeline.

> 1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.

Author: Yep, `Memory Utilization` and `CPU Utilization`.

> 1. Click the **Workers** tab to see the Workers' memory and CPU utilization, traffic stats, and any errors.

Reviewer: Unfortunate ;)

## View the status of your pipeline components

To view metrics for a source, process, or destination:

1. Navigate to [Observability Pipelines][1].
1. Select your pipeline.

> 1. Click the cog next to the source's, processor's, or destination's name, and then select **View details** to see health graphs for that component.

Reviewer: ```suggestion
1. Click the cog next to the source's, processor's, or destination's name, then select **View details**. Datadog displays health graphs for the component you selected.
```

> 1. Click the export icon on a graph if you want to export it to an [incident][2], [dashboard][3], or [notebook][4]. The exported graph shows you how the metric is grouped by the specific pipeline and component tags.

Reviewer: ```suggestion
1. If you want to export a graph to an [incident][2], [dashboard][3], or [notebook][4], click the export icon on the graph. The exported graph shows that the metric is grouped by the specific pipeline and component tags.
```

## Out-of-the-box monitors

> To see the available out-of-the-box monitors:

Reviewer: ```suggestion
To see available out-of-the-box monitors:
```

1. Navigate to [Observability Pipelines][1].
1. Click **Enable monitors** in the **Monitors** column for your pipeline.

> 1. Click **Enable** to set up a monitor for that use case.

Reviewer: I'm not sure what "that use case" is referring to here, can you elaborate?

Author: When you click Enable Monitors, a side panel opens up with a list of recommended monitors for different scenarios to monitor, like components are dropping data, components are producing errors, high CPU usage, etc that you can enable. So those scenarios are the "use case" I'm referring to. In demo org: https://app.datadoghq.com/observability-pipelines?isEditor=false&pageIndex=12&sort=created%2Cdesc

> 1. Click **Enable** to set up a monitor for that use case.<br>

Reviewer: Maybe instead of "that use case" it could be something like "one of the suggested use cases"?

Also I see **Start** and not **Enable**, are we seeing different things?

Author: Ah, I see **Start** too. Updated that and also used your suggestion regarding "that use case".

> 1. Click **Enable** to set up a monitor for that use case.
> 1. The new metric monitor page is configured based on the use case you selected. You can update the configuration to further customize it. See the [Metric monitor documentation][3] for more information.

Reviewer: ```suggestion
1. Click **Enable** to set up a monitor for that use case.<br>
    The new metric monitor page is configured based on the use case you selected. You can update the configuration to further customize it. See the [Metric monitor documentation][3] for more information.
```
